drivers/wifi/nxp/Kconfig.nxp

```diff
@@ -469,6 +469,12 @@ config NXP_WIFI_11R
 	help
 	  This option enables the use of 802.11r support.
 
+config NXP_WIFI_RTS_THRESHOLD
```

Comment:
do we really need a build time option for the feature? Can't we just use default as `2347` or `-1`?

Reply:
We have macro in nxp driver code to enable/disable RTS threshold setting, so here add NXP_WIFI_RTS_THRESHOLD to co-work with driver.